test/Frontend/crash.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: echo %s > %t.filelist.txt
```

Comment:
I know that the following isn't our typical practice, but as a noob, it would help me to add a one sentence comment to this test describing what it does.